src/pharmpy/modeling/plots.py

```diff
@@ -399,17 +473,15 @@ def _dv_vs_anypred(model, predictions, predcol_name, predcol_descr):
         tooltip=(idv,),
     )
 
-    line = _identity_line(
-        df[predcol_name].min(), df[predcol_name].max(), df[dv].min(), df[dv].max()
-    )
-
-    layer = chart + _smooth(chart, predcol_name, dv) + line
+    layer = chart + _smooth(chart, predcol_name, dv) + _identity_line()
     layer = layer.configure_point(size=60)
 
     return layer
 
 
-def _dv_vs_anypred_stratify(model, predictions, predcol_name, predcol_descr, strat=None, bins=8):
+def _dv_vs_anypred_stratify(
```

Comment:
Would it be possible to share code between this and the non-stratification function or are they too different?